### PR TITLE
Write logout route and logout controller

### PR DESCRIPTION
**Pull request by mAbadsa**
Relates #40

**Logout route**

- [x] 1. In this branch, we build a logout process to clear auth token from the cookies.
- [x] 2. This process consists of a logout route and a logout controller.
- [x] 3. We use the Expressjs framework and use the pre-built Router method.

### CODE REVIEW

**Review by yasmeenattallah — DISMISSED** [review]

File: server/controller/routes/user/logout.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const logout = (req, res, next) => {
+  try {
+    return res
```
Comment (yasmeenattallah) [inline server/controller/routes/user/logout.js:3]: should you use async .. await

**Review by AdhamHaisami — DISMISSED** [review]

File: server/router/index.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
 const router = require('express').Router();
 
+const { user } = require('./user');
+
 router.get('/', (req, res) => {
   res.json({
     message: 'Hello from house-hunting app',
     statusCode: 200,
   });
 });
```
Comment (mariamisa) [inline server/router/index.js:11]: delete this test route

**Review by mariamisa — CHANGES_REQUESTED** [review]

File: server/controller/routes/user/index.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+const logout = require('./user/logout');
```
Comment (mariamisa) [inline server/controller/routes/user/index.js:1]: this path not true

File: server/router/user.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
 const router = require('express').Router();
 
+const { logout } = require('../controller/routes/user/index');
```
Comment (mariamisa) [inline server/router/user.js:3]: remove index from require

**Review by mariamisa — CHANGES_REQUESTED** [review]

**Review by mariamisa — APPROVED** [review]

**Pull request merged**